idp-service/src/main/java/io/mosip/idp/services/ClientManagementServiceImpl.java

```diff
@@ -5,25 +5,195 @@
  */
 package io.mosip.idp.services;
 
-import io.mosip.idp.core.dto.ClientDetailRequest;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.mosip.idp.core.dto.ClientDetailCreateRequest;
 import io.mosip.idp.core.dto.ClientDetailResponse;
+import io.mosip.idp.core.dto.ClientDetailUpdateRequest;
+import io.mosip.idp.core.exception.IdPException;
 import io.mosip.idp.core.spi.ClientManagementService;
+import io.mosip.idp.core.util.ErrorConstants;
+import io.mosip.idp.domain.ClientDetail;
+import io.mosip.idp.repositories.ClientDetailRepository;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
+import java.security.KeyFactory;
+import java.security.spec.X509EncodedKeySpec;
+import java.util.Base64;
+
 @Service
 public class ClientManagementServiceImpl implements ClientManagementService {
+    public static final String TAG = ClientManagementServiceImpl.class.getSimpleName();
+
+    @Autowired
+    ClientDetailRepository clientDetailRepository;
+
+    ObjectMapper mapper = new ObjectMapper();
```

Comment:
Instance of object mapper is available in context, so need to auto-wire object mapper instance